components/review-item.js

```diff
@@ -0,0 +1,142 @@
+      throw new Error('The "name" attribute is required.');
+    }
+  }
+
+  render() {
+    this.attachShadow({ mode: "open" });
+    this.shadowRoot.innerHTML = this.createCss() + this.createHtml();
+  }
+
+  createCss() {
+    return css`
+      .review-item {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        border-radius: 10px;
+        padding: 39px 20px;
+        background-color: var(--bg-300);
+        color: var(--text-900);
+      }
+      .review-item blockquote,
+      figure {
+        margin: 0;
+      }
+      .review-img {
+        width: 104px;
+        height: 104px;
+        flex-shrink: 0;
+        object-fit: cover;
+      }
+      .review-img figure > img {
+        width: 100%;
+        height: 100%;
+        padding-top: 7px;
+        border-radius: 48%;
+        background-color: #ffffff;
+      }
+      .review-content {
+        display: flex;
+        flex-direction: column-reverse;
+        align-items: center;
+        text-align: center;
+        margin-top: 10px;
+      }
+      .review-content blockquote {
+        font-size: 14px;
+        margin-top: 17px;
+        word-break: keep-all;
+      }
+      .review-content figcaption {
+        width: 100%;
+        font-size: 16px;
+        text-align: center;
+      }
+
+      /* Small devices such as large phones (640px and up) */
+      @media only screen and (min-width: 640px) {
+        /* TODO: Check if the media query guideline supports small screens or not. */
+      }
+
+      /* Medium devices such as tablets (768px and up) */
+      @media only screen and (min-width: 768px) {
+        .review-item {
+          padding: 37px 20px;
+        }
+        .review-content blockquote {
+          margin-top: 16px;
+        }
+      }
+
+      /* Large devices such as laptops (1024px and up) */
+      @media only screen and (min-width: 1024px) {
+        .review-item {
+          padding: 41px 51px 31px;
+          flex-direction: row;
+          align-items: flex-start;
+        }
+        .review-img {
+          margin-right: 31px;
+        }
+        .review-content {
+          flex-direction: column;
+          margin-top: 0;
+          text-align: left;
+        }
+        .review-content blockquote {
+          margin-top: 0;
+          margin-bottom: 14px;
+          word-break: unset;
+        }
+        .review-content figcaption {
+          text-align: right;
+        }
+      }
+
+      /* Largest devices such as desktops (1280px and up) */
+      @media only screen and (min-width: 1280px) {
+        /* TODO: Add after checking the media query guideline. */
+      }
+    `;
+  }
+
+  createHtml() {
+    const img = this.getAttribute("img");
+    const text = this.getAttribute("text");
+    const name = this.getAttribute("name");
+
+    return html`
+      <article class="review-item">
+        <section class="review-img">
+          <figure><img src="${img}" alt="Reviewer" /></figure>
```

Comment:
alt의 용도에 맞게 해당 이미지가 어떤 역할을 하는 이미지인지 알려주면 좋을 것 같습니다.

```suggestion
          <figure><img src="${img}" alt="reviewer profile image" /></figure>
```

Reply:
동의합니다 진짜 alt 속성을 활용하는 분들 입장을 좀 더 생각해보면, 명확할수록 더 좋을 것 같습니다.
Author profile image로 변경완료하였습니다!